### ISSUE TITLE
`send --text -` reads in all the text before sending

### ISSUE THREAD

**Issue by joshtriplett** [warner/magic-wormhole#311]
I wanted to stream over a pipe, so I tried piping into `wormhole send --text -`, but that seems to read in all the data first, so it ran out of memory.

**Comment by warner** [comment]
Yeah, sorry, the send-text protocol currently only sends a single message, and doesn't support streams.

When we get the "Dilation" protocol (#312) implemented, then streams should be pretty easy, and `--text -` won't need to buffer everything first.
